.travis/run_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
can you place this script into utils? (run_onnxifi_tests.sh or something)

Reply:
how hard would it be to make that actual CMake target? so that one can run it with: ninja onnxifi_tests.

Reply:
So you want the excutable gtest to be a cmake target?

Reply:
as talked offline, stand alone bash/python script would be good enough as long as it could be executed locally as well.